client/hwlib/src/constants.rs

```diff
@@ -23,6 +23,9 @@ pub const PROC_CPUINFO_FILE_PATH: &str = "/proc/cpuinfo";
 pub const PROC_DEVICE_TREE_DIR_PATH: &str = "/proc/device-tree/";
 pub const PROC_VERSION_FILE_PATH: &str = "/proc/version";
 
+// The os-release file path is set by a build-time environment variable
```

Comment:
```suggestion
// The os-release file path is set during the build time
```

I think it’s important to indicate that it cannot be modified by a user (cannot be changed during the runtime), but feel free to ignore id you think that’s fine

Reply:
Sure, that's a reasonable change, and makes the comment more definitive